util/cmd.go

```diff
@@ -82,8 +83,9 @@ func execCommandOutputWithTimeout(cmd string, args []string, stdinArgs []string,
 
 // ExecCommandOutputWithTimeout  executes ExecCommandOutput with the specified timeout
 func ExecCommandOutputWithTimeout(cmd string, args []string, timeout int) (string, int, error) {
-	return execCommandOutputWithTimeout(cmd, args, []string{}, defaultTimeout)
```

Comment:
This method(ExecCommandOutputWithTimeout) is getting called in only three places with right timeout defined for the operations
linux/mount.go - timeout 300
linux/os.go - timeout 180
linux/device.go - timeout  15

In some cases mkfs command is taking more than 60 seconds and the driver is breaking with the timeout error and device serail number not found issue. Whats happening is, timeout mentioned in mount.go(300 seconds) is not getting passed to the   "ExecCommandOutputWithTimeout" method because of the parameter error in this method. 

"ExecCommandOutpu"t method can be used incase caller doesnt want to set the timeout. This method is using the default timeout mentioned in the cmd.go

Reply:
@sijeesh what @raunakkumar discovered was that the interface changed in https://github.com/hpe-storage/common-host-libs/commit/7b80c75521e1a06c81f1e7264474fda30939dcfa and needs to be reverted.